packages/editor-ui/src/plugins/codemirror/completions/nativesAutocompleteDocs/luxon.static.docs.ts

```diff
@@ -192,46 +349,74 @@ export const luxonStaticDocs: Required<NativeDoc> = {
 				name: 'fromString',
 				docURL: 'https://moment.github.io/luxon/api-docs/index.html#datetimefromstring',
 				returnType: 'DateTime',
+				hidden: true,
 				args: [
 					{ name: 'text', type: 'string' },
 					{ name: 'fmt', type: 'string' },
-					{ name: 'options?', type: 'object' },
+					{ name: 'options', optional: true, type: 'Object' },
 				],
 			},
 		},
 		fromStringExplain: {
 			doc: {
 				name: 'fromStringExplain',
 				docURL: 'https://moment.github.io/luxon/api-docs/index.html#datetimefromstringexplain',
-				returnType: 'object',
+				returnType: 'Object',
+				hidden: true,
 				args: [
 					{ name: 'text', type: 'string' },
 					{ name: 'fmt', type: 'string' },
-					{ name: 'options?', type: 'object' },
+					{ name: 'options', optional: true, type: 'Object' },
 				],
 			},
 		},
 		max: {
 			doc: {
 				name: 'max',
+				description: i18n.baseText('codeNodeEditor.completer.luxon.dateTimeStaticMethods.max'),
 				docURL: 'https://moment.github.io/luxon/api-docs/index.html#datetimemax',
-				returnType: 'DateTime|undefined',
+				returnType: 'DateTime | undefined',
```

Comment:
> [DateTime](https://moment.github.io/luxon/api-docs/index.html#datetime): the max `DateTime`, or `undefined` if called with no argument

What's the use case of calling with no arg? If none maybe we shouldn't document `undefined`. Same below.

Reply:
Yeah good call, I don't think there is any